config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/items", to: "items#index"
+
+
+  get "/customers/:id", to: "customers#show"
+
```

Comment:
Based on the comments I left on your form and controller, you'd also want to add this one when refactoring to follow RESTful/SRP conventions:
`post "/customers/:customer_id/items", to: "customer_items#create"`

Notice the controller is different -- this is using a joins controller rather than the same customers controller!